sources/chargen.js

```diff
@@ -935,9 +936,13 @@ $(document).ready(function() {
     setTimeout(nextFrame, 1000 / 8);
   }
 
-  function updatePreviewLink(imageLink) {
+  function updatePreviewLink(imageLink, customWalkAnimation) {
     const { directory, file } = splitFilePath(imageLink);
-    imageLink = `${directory}/walk/${file}`;
+    if (customWalkAnimation) {
+      imageLink = `${directory}/${file}`
```

Comment:
This special case is used for animations where there is a walk but we need to preview from walk_128 because walk doesn't have the oversize weapons.